neighbour-model/src/main/java/no/vegvesen/ixn/federation/capability/CapabilityMatcher.java

```diff
@@ -197,11 +197,21 @@ private static String evaluateQuadTreeMatch(String selector, Set<String> capabil
 				match = matchDenm((DenmCapability) capability, selectorFilter);
 			} else if (capability instanceof IvimCapability) {
 				match = matchIvi((IvimCapability) capability, selectorFilter);
+			} else if (capability instanceof SpatemCapability) {
```

Comment:
Maybe go through the code and see if it's possible to get rid of some of the duplications?